### DISCUSSION TITLE
Integration with Bazel

### DISCUSSION THREAD

**Discussion by Mateusz-Krajewski** [Kistler-Group/sdbus-cpp#366] [Ideas]
Anyone rewrite this library to Bazel or have idea how to add it to Bazel project?

**Comment by sangelovic** [comment]
I have no idea.

**Comment by sangelovic** [comment]
Perhaps if Bazel supports something like an external project feature, then one can build sdbus-c++ with cmake invoked from a Bazel-based downstream project.
